File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/OperatingSystem.scala
package io.swagger.server.model

import java.math.BigDecimal

/**
 * An operating system option available for server provisioning.
 *
 * @param id 
 * @param short_desc 
 * @param monthly_price 
 */
case class OperatingSystem (
  id: Option[Int],
  short_desc: Option[String],
  monthly_price: Option[BigDecimal]
)

File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/OperatingSystem.js
goog.provide('API.Client.OperatingSystem');

/**
 * An operating system option available for server provisioning.
 * @record
 */
API.Client.OperatingSystem = function() {}

/**
 * @type {!number}
 * @export
 */
API.Client.OperatingSystem.prototype.id;

/**
 * @type {!string}
 * @export
 */
API.Client.OperatingSystem.prototype.shortDesc;

/**
 * @type {!number}
 * @export
 */
API.Client.OperatingSystem.prototype.monthlyPrice;

